ministryofjustice/correspondence_tool_staff
# CT-2439 Add translations for clearing response

## Description
Add translations and remove strings within Clear responses form
 
## Self-review checklist

 
### Screenshots
![image](https://user-images.githubusercontent.com/22935203/69075020-79ffd380-0a28-11ea-8ec9-fb300f82999d.png)

 
### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2439
 
### Deployment
n/a
 
### Manual testing instructions
Create a case with a response pending Press Office Clearance, then log in as Press Office and clear the response, view the form to ensure it all looks Ok.
